### PR TITLE
Add CODEOWNERS and master PR version check and fix functionality

### PR DESCRIPTION
**Pull request by ddbruce**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -5,6 +5,19 @@ require("dotenv").config();
 const { app } = require("./utilities/bolt.js");
 const { processMessage } = require("./utilities/helperFunctions.js");
 
-app.message(async ({ message }) => {
-  await processMessage(message);
+//globals
+
+//for dev
+let DEV;
```
Comment (lramos15) [inline src/server.js]: How about just setting dev = to this

File: src/server.js [hunk 2]
```diff
@@ -5,6 +5,19 @@ require("dotenv").config();
...
+  DEV = true;
+} else {
+  DEV = false;
+}
+
+app.message(({ message, message: { subtype } }) => {
+  if (DEV) {
+    console.log(message);
+  }
+  if (subtype != "message_deleted") processMessage(message);
```
Comment (lramos15) [inline src/server.js]: Break this up onto three lines for readability

File: src/utilities/helperFunctions.js [hunk 3]
```diff
@@ -17,22 +17,35 @@ const MOD_USERGROUP_ID = process.env.MOD_USERGROUP_ID;
 const ADMIN_USERGROUP_ID = process.env.ADMIN_USERGROUP_ID;
 const MONITORED_CHANNELS = process.env.MONITORED_CHANNELS.split(",");
 
-const isModerator = async (user) => {
+//for dev
+let DEV;
```
Comment (lramos15) [inline src/utilities/helperFunctions.js]: Duplicate code. Make into a utility function

File: src/utilities/helperFunctions.js [hunk 4]
```diff
@@ -17,22 +17,35 @@ const MOD_USERGROUP_ID = process.env.MOD_USERGROUP_ID;
...
+} else {
+  DEV = false;
+}
+
+const isNotModerator = async (user) => {
   const { users: adminUsers } = await list({
     token: TOKEN,
     usergroup: ADMIN_USERGROUP_ID,
   });
-  if (adminUsers.includes(user)) return true;
```
Comment (lramos15) [inline src/utilities/helperFunctions.js:25]: I'm not a fan of single line ifs

**Review by lramos15 — CHANGES_REQUESTED** [review]
Small nits

File: src/server.js (hunk 1 above)
Comment (ddbruce) [reply src/server.js]: There are a few ways to skin this cat. If you do a `const DEV = ` whatever in the if statement, it's scoped and can't be accessed in the rest of the code. I could do a ternary or something 🤷‍♂️

File: src/utilities/helperFunctions.js (hunk 4 above)
Comment (ddbruce) [reply src/utilities/helperFunctions.js:25]: 😭

**Review by lramos15 — APPROVED** [review]
😊

**Pull request merged**